src/samples/ChildBenefitsClaim/UserPortal.tsx

```diff
@@ -49,12 +55,62 @@ export default function UserPortal(props) {
                 <h2 className='govuk-heading-m' id='subsection-title'>
                   {t('MAKE_A_CLAIM')}
                 </h2>
-                <p className='govuk-body'>{t('USE_THIS_SERVICE')}</p>
-                <p className='govuk-body'>{t('WE_MAY_CALL_YOU')}</p>
+                {(() => {
+                  if (showBeginNewClaimButton && beginNewClaimButtonForInProgressClaims) {
```

Comment:
Could we rename these?
When I first read it I would assume showBeginNewClaimButton  being true would show the claims button.

Reply:
And do these need to be in a function?

Reply:
> And do these need to be in a function?

Hi Cart, how are we going to include if-else statements inside JSX? I have used IIFE (Immediately Invoked Function Expression) . Any other suggestions from you?

Reply:
> Could we rename these? When I first read it I would assume showBeginNewClaimButton being true would show the claims button.

have changed it now